Formula 1/ViewControllers/CircuitsCollectionView.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  CircuitsCollectionView.swift
+//  Formula 1
+//
+//  Created by Gio on 11/13/19.
+//  Copyright © 2019 Gio. All rights reserved.
+//
+
+import UIKit
+
+final class ConstructorsFlowLayout: UICollectionViewFlowLayout {
+    override func prepare() {
```

Comment:
So I had this in the wrong file - I meant to include it inside ConstructorsCV. I am using this single flowLayout for both collection views until i create the compositional layout, so this is very temporary. I opted to keep it default encapsulation and create an instance of it until I can replace it instead of copying and pasting the same code

Reply:
I still would crate its own file

Reply:
I created a solution for now